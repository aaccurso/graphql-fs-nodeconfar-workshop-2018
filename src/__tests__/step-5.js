const { graphql } = require('graphql');
const schema = require('../schema');
const { listRecursive } = require('../queries');

describe('step 5', () => {
  it('should query list recursively', async () => {
    expect(await graphql({ schema, source: listRecursive })).toMatchSnapshot();
  });
});
