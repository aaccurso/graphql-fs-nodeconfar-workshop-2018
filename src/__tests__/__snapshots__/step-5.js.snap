// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`step 5 should query list recursively 1`] = `
Object {
  "errors": Array [
    [GraphQLError: Syntax Error: Unexpected Name "ls"],
  ],
}
`;
